### ISSUE TITLE
Begin evaluating the pattern proposal for prefilled forms

### ISSUE THREAD

**Issue by jaclinec** [uswds/uswds-team#400]
We received a pattern proposal for "[Help users to know when their information is prefilled](https://github.com/uswds/uswds/discussions/6053)" and the next step is to begin a first pass UX and accessibility evaluation of the proposal.

We will evaluate the proposal based on the criteria: 

- Does it support a WCAG requirement or recommendation?
- Does it effectively address a user need?
- Is there an existing USWDS solution to fulfill the need?
- Is this component needed by the USWDS community?
- Do the usability and accessibility benefits outweigh the potential risks of using the component?

**Comment by amycole501** [comment]
I've performed the lion's share of my research and evaluation. Documentation and findings are here https://docs.google.com/document/d/1grmsmtEk0wwRcnanOfmLqEfUZKwdqbF7dY0fZLpYMN8/edit?usp=sharing

**Comment by jaclinec** [comment]
I've completed my review as well and documented them in that Google doc Amy shared. Will next discuss with Amy and send a few questions to the team who submitted the proposal in the discussion.

**Comment by jaclinec** [comment]
I created the [Pre-fill pattern final evaluation document](https://docs.google.com/document/d/1oIjHWX_0ev1vNegfFY0HizH0111thqGQ5kyx8Rn6sVY/edit?usp=sharing) based on our investigation. We recommend moving forward with the proposal and the next considerations are how detailed we want to be in the design implementation guidance of the pattern and what that guidance might be (needs discussion).

I also posted a couple of questions in the proposal discussion and waiting to hear back. This part of the work is done though.
